feature/experimental/bgp/otg_tests/bgp_tcp_mss_path_mtu/README.md

```diff
@@ -0,0 +1,42 @@
+# RT-1.21: BGP TCP MSS and PMTUD 
+
+## Summary
+
+BGP TCP MSS and PMTUD
+
+## Topology
+
+"Virtual host on ATE" <--> ATE (port1) <--> DUT
+
+## Procedure
+
+*   Establish BGP sessions between:
+    *   ATE (port-1) ---  eBGP-IPv4/IPv6 ---- DUT 
+    *   Virtual host on ATE ---- iBGP IPv4 ---- DUT.
```

Comment:
Since MTU on virtual host not supported, this needs to be a 2xDUT test. What is the plan to close on this test?

Reply:
I have updated README with 2 DUT .  Thank you.